xep-0050.xml

```diff
@@ -520,17 +520,19 @@
       </li>
       <li>If there is an &lt;actions/&gt; element, the user-agent usually uses a multi-stage dialog or view, such as a wizard.
         <ul>
-          <li>The action "execute" is always allowed, and is equivalent to the action "next".</li>
+          <li>The action "execute" is always allowed, and is equivalent to the action "next". A form that has an &lt;actions/&gt; element, no "execute" attribute, and for which &lt;next/&gt; is not an available action is therefore invalid.</li>
           <li>The "prev" action is typically the "back" or "previous" button or option in a wizard. If &lt;prev/&gt; is not contained by the &lt;actions/&gt;, it is disabled.</li>
           <li>The "next" action is typically the "next" button or option in a wizard. If &lt;next/&gt; is not contained by the &lt;actions/&gt;, it is disabled.</li>
           <li>The "complete" action is typically the "finish" or "done" button or option in a wizard. If &lt;complete/&gt; is not contained by the &lt;actions/&gt;, it is disabled.</li>
           <li>If the &lt;actions/&gt; possesses the "execute" attribute, that value is the default button or option. If the &lt;actions/&gt; does not possess the "execute" attribute, there is no default button or option.</li>
+          <li>As the "execute" command is equivalent to another command ("next" in the absence of an "execute" attribute), it is not suggested that a user-agent shows both the "execute" and the command that "execute" is an alias of in their interface.</li>
         </ul>
       </li>
     </ol>
     <p>Responders SHOULD use the following guidelines when providing &lt;actions/&gt;:</p>
     <ul>
       <li>The "execute" attribute SHOULD NOT specify a value that does not match one of the allowed actions.</li>
+      <li>As user-agents treat the absence of an "execute" attribute as equivalent to "next", when the &lt;actions/&gt; element is provided, it is invalid to return a command that has no "execute" attribute, an actions element, and no &lt;next/&gt; action</li>
```

Comment:
It appears that this just duplicates what is added above. If so, I'd suggest to only state the requirement once.